tests/test_includes.py

```diff
@@ -153,3 +157,11 @@ def test_monorepo_runs_each_task_with_expected_cwd(
     else:
         assert result.stdout.endswith("/tests/fixtures/monorepo_project/subproject_2\n")
     assert result.stderr == ""
+
+    result = run_poe_subproc("get_cwd_3", project="monorepo", cwd=projects["example"])
+    assert result.capture == "Poe => import os; print(os.getcwd())\n"
+    if is_windows:
+        assert result.stdout.endswith("poethepoet\\tests\\fixtures\\example_project\n")
+    else:
+        assert result.stdout.endswith("poethepoet/tests/fixtures/example_project\n")
```

Comment:
I recently changed the examples in this test because it makes the test dependent on the name of the checkout directory.

```suggestion
        assert result.stdout.endswith("/tests/fixtures/example_project\n")
```